Author: Tom Jampen <tom@cryptography.ch>
Description: Fix plugin path to use debian's request-tracker plugin directory.

diff -Naurp a/inc/Module/Install/RTx.pm b/inc/Module/Install/RTx.pm
--- a/inc/Module/Install/RTx.pm	2012-05-10 16:22:06.166168825 +0200
+++ b/inc/Module/Install/RTx.pm	2012-06-11 12:51:31.879609814 +0200
@@ -38,8 +38,9 @@ sub RTx {
     @ARGV = grep { /PREFIX=(.*)/ ? ( ( $prefix = $1 ), 0 ) : 1 } @ARGV;
 
     if ($prefix) {
-        $RT::LocalPath = $prefix;
-        $INC{'RT.pm'} = "$RT::LocalPath/lib/RT.pm";
+        push @INC, "$prefix";
+        require RT;
+        $RT::LocalPluginPath = "$prefix/plugins";
     } else {
         local @INC = (
             $ENV{RTHOME} ? ( $ENV{RTHOME}, "$ENV{RTHOME}/lib" ) : (),

